mypy/dmypy.py

```diff
@@ -0,0 +1,585 @@
+"""Client for mypy daemon mode.
+
+Highly experimental!  Only supports UNIX-like systems.
+
+This manages a daemon process which keeps useful state in memory
+rather than having to read it back from disk on each run.
+"""
+
+import argparse
+import gc
+import io
+import json
+import os
+import signal
+import socket
+import sys
+import time
+
+from typing import Any, Callable, Dict, List, Mapping, Optional, Sequence, Tuple, TypeVar
+
+import psutil  # type: ignore  # It's not in typeshed yet
+
+import mypy.build
+import mypy.errors
+import mypy.main
```

Comment:
Eventually we could import most mypy modules lazily so that the client would run quicker in case the server is already running.